config/settings.yml

```diff
@@ -5,7 +5,7 @@ purl_conn_timeout: 2
 purl_feedback_email: ''
 valid_purl_url: <%= /https?:\/\/purl\.stanford\.edu\/*/ %>
 stacks_url: 'https://stacks.stanford.edu'
-iiif_stacks_url: 'https://stacks.stanford.edu'
+iiif_stacks_url: 'https://stacks.stanford.edu' # unused???
```

Comment:
👍 I'm not sure we need a ticket officially for this.  But I do like being more careful to document these things in the future (similarly to how we do for the Gemfile).